action.yml

```diff
@@ -13,6 +13,11 @@ inputs:
     default: medium
   ignore_list:
     description: List of CVE IDs to ignore in the vulnerability findings.
+  error_missed_ignores:
```

Comment:
Please also update the readme :)

Reply:
I've updated the PR with the recommended changes; as well have updated the readme to reflect this new parameter.